crates/revm/Cargo.toml

```diff
@@ -21,10 +21,7 @@ serde = { version = "1.0", features = ["derive", "rc"], optional = true }
 serde_json = { version = "1.0", features = ["preserve_order"], optional = true }
 
 # ethersdb
-tokio = { version = "1.32", features = [
-    "rt-multi-thread",
-    "macros",
-], optional = true }
+tokio = { version = "1.32", features = ["rt-multi-thread", "macros"], optional = true }
```

Comment:
I think we have two different formatter for toml. I am using https://taplo.tamasfe.dev/ but not sure if I was the one formatting this

Reply:
I am too, I have it configured to 100 column width, so that might be it

Reply:
Can you use default value and revert it back to be consistent? Maybe we should add local vscode setup, and change it there.